### ISSUE TITLE
Game can only download maps if installed to C: drive

### ISSUE THREAD

**Issue by angelicself** [actionquake/distrib#100]
The Windows version of the game downloads maps to AppData\Local\\.aqtion\action\maps, regardless of where the game folder is located. If the game folder is on a different drive (eg, "D:\Games\SteamLibrary\steamapps\common\AQtion\"), q2pro doesn't have permission to download there. Running the game as administrator does not solve the issue.

**Comment by darkshade9** [comment]
Correct, that's because the files are going to the local `action` directory instead of %LOCALAPPDATA%.  Currently, Steam overwrites all files in the `action` directory every time it pushes an update, which deletes all downloaded maps, textures, and your user settings (config.cfg).  This is a workaround for this problem, but I am also looking into the possibility for Steam to not overwrite any content it is not aware of, persisting your content and settings across updates.  I'll update this thread with any new information

**Comment by darkshade9** [comment]
Try either one of these binaries @angelicself
[q2pro.zip](https://github.com/actionquake/distrib/files/8733973/q2pro.zip)

**Comment by angelicself** [comment]
Sorry, didn't think to check this. That one downloads to the local `action` directory, which I'm guessing is what you wanted it to do.

**Comment by darkshade9** [comment]
Yep, wanted to test it out, next official release will have this as standard.
